app/ts/components/subcomponents/Spinner.tsx

```diff
@@ -1,11 +1,11 @@
-export function Spinner({ height } : { height: string }) {
+export function Spinner({ height, color } : { height: string, color?: string }) {
 	return (
 		<svg
-			style = { { height, margin: 'auto'  } }
+			style = { { height, margin: 'auto'} }
 			class = 'spinner'
 			viewBox = '0 0 100 100'
 			xmlns = 'http://www.w3.org/2000/svg'>
-				<circle cx = '50' cy = '50' r = '45'/>
+				<circle cx = '50' cy = '50' r = '45' style = { { ...color !== undefined ? { stroke: color } : {}  } }/>
```

Comment:
syntax is valid but looks like `color` is being spread, wrapping for clarity
```suggestion
				<circle cx = '50' cy = '50' r = '45' style = { { ...( color !== undefined ? { stroke: color } : {} )  } }/>
```